### PR TITLE
fix: [] Wrap inline nodes inside of a table cell in paragraph node

### PR DESCRIPTION
**Pull request by colomolo**
When there are multiple inline nodes in a table cell it wraps each inline node in its own paragraph node, which is incorrect behavior. Wrapping consecutive inline nodes with single paragraph.

### CODE REVIEW

File: packages/rich-text-from-markdown/src/index.ts [hunk 1]
```diff
@@ -328,6 +337,7 @@ function expandParagraphWithInlineImages(node: MarkdownNode): MarkdownNode[] {
 // so we must hoist them out before transforming to rich text.
 function prepareMdAST(ast: MarkdownTree): MarkdownNode {
   function prepareASTNodeChildren(node: MarkdownNode): MarkdownNode {
+    console.log({ node: JSON.stringify(node, null, 2) });
```
Comment (chrishelgert) [inline packages/rich-text-from-markdown/src/index.ts:340]: 👀

File: packages/rich-text-from-markdown/src/index.ts [hunk 2]
```diff
@@ -357,6 +367,7 @@ export async function richTextFromMarkdown(
 ): Promise<Document> {
   const processor = unified().use(markdown).use(gfm);
   const tree = processor.parse(md);
+  // console.log({tree: JSON.stringify(tree, null, 2)});
```
Comment (chrishelgert) [inline packages/rich-text-from-markdown/src/index.ts:370]: 👀

File: packages/rich-text-from-markdown/src/index.ts (hunk 1 above)
Comment (colomolo) [reply packages/rich-text-from-markdown/src/index.ts]: whoopsie

**Review by chrishelgert — APPROVED** [review]
LGTM

**Review by z0al — APPROVED** [review]

**Pull request merged**